eclass-writing/text.xml

```diff
@@ -75,7 +76,9 @@ If there is an existing maintainer for an eclass (this is usually the case), you
 
 <p>
 It is not usually necessary to email the gentoo-dev list before making changes
-to a non-general eclass which you maintain. Use common sense here.
+to a non-general eclass which you maintain. For example, an eclass that is used
```

Comment:
I would go really specific here instead of giving an example in an example:

```
For example, the <c>apache2</c> eclass is only used by the <c>www-servers/apache</c> package, and thus does not typically require changes to be emailed for review.
```